### DISCUSSION TITLE
Robot aspirapolvere IKOHS CREATE s18

### DISCUSSION THREAD

**Discussion by mefrasa** [indomus/forum#344] [GENERALE - Discussioni generiche]
Buongiorno, 
qualcuno di voi è riuscito ad integrare il robot aspirapolvere in HA?
Io vorrei integrare il IKOHS s18

Grazie

**Comment by indomus** [comment]
(Se non trovi un'integrazione funzionante, alla fine [puoi fare così.](https://indomus.it/progetti/domotizzare-un-robot-aspirapolvere-non-domotico-con-broadlink-e-home-assistant/))

**Reply by mefrasa** [reply]
Grazie, avevo visto questa guida, ma essendo un robot wi-fi volevo evitare di aggiungere altri elementi.
Stavo vedendo il componente https://indomus.it/componenti/componente-home-assistant-vacuum-aspirapolvere/ 
ma non trovo come fare la configurazione e se la marca è supportata o meno.

**Reply by indomus** [reply]
quello che indichi è il componente "padre", quel che ti serve è la piattaforma "figlia" adeguata ([leggi qui](https://indomus.it/formazione/comprendere-la-logica-componente-piattaforma-entita-di-home-assistant/)).

per il tuo [non c'è niente di ufficiale](https://www.home-assistant.io/integrations#search/IKOHS), devi semmai cercare un custom component esterno che lo supporti, ma da una rapida occhiata pare non ci sia nulla.
